riverdriver/riverdatabasesql/go.mod

```diff
@@ -5,6 +5,7 @@ go 1.21
 toolchain go1.23.0
 
 require (
+	github.com/jackc/pgx/v5 v5.7.1
```

Comment:
Hmm, I guess this is what I get for using `pgtype.bits` for the bitmask column. I'd be shocked if pq properly supports that, thoughts on what to do?